CITATION.cff

```diff
@@ -0,0 +1,32 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Fluxtower Above-ground Biomass Intercomparison
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Eric R.
+    family-names: Scott
+    email: ericrscott@arizona.edu
+    affiliation: >-
+      University of Arizona, Communications & Cyber
+      Technologies
+    orcid: 'https://orcid.org/0000-0002-7430-7879'
+  - given-names: Lindsey M.
+    family-names: Bell
+    affiliation: >-
+      University of Arizona, School of Natural Resources &
+      The Environment
```

Comment:
Just made an orcid to add: https://orcid.org/0009-0001-2325-9977

All other info is correct. Thanks, Eric!